client/plots/hierCluster.js

```diff
@@ -238,13 +233,13 @@ export class HierCluster extends Matrix {
 		}
 		const absMax = Math.min(hc.zScoreCap, Math.max(...extent(globalMinMaxes).map(Math.abs)))
 		const [min, max] = [-absMax, absMax]
-		// what's purpose of assigning this.geneExpValues{}, to signal something to matrix code?
-		this.geneExpValues = { scale, min, max }
+		// what's purpose of assigning this.hierClusterValues{}, to signal something to matrix code?
+		this.hierClusterValues = { scale, min, max }
 	}
 
 	getValueColor(value) {
 		const zScoreCap = this.settings.hierCluster.zScoreCap
-		return this.geneExpValues.scale((value - -zScoreCap) / (zScoreCap * 2))
```

Comment:
did you meant -- or +?

Reply:
here means two minus, so we could set the range to be (-zScoreCap, zScoreCap)